src/pages/ServiceDetails/ServiceInfo/ServiceInfoDestinationRules.tsx

```diff
@@ -153,32 +83,72 @@ class ServiceInfoDestinationRules extends React.Component<ServiceInfoDestination
     );
   }
 
-  rows() {
-    return (this.props.destinationRules || []).map((destinationRule, vsIdx) => {
-      return {
-        id: vsIdx,
-        name: this.overviewLink(destinationRule),
-        status: (
-          <ValidationSummary
-            id={vsIdx + '-config-validation'}
-            validations={this.hasValidations(destinationRule) ? [this.validation(destinationRule)] : []}
-          />
-        ),
-        trafficPolicy: destinationRule.spec.trafficPolicy ? (
-          <DetailObject name="" detail={destinationRule.spec.trafficPolicy} />
-        ) : (
-          'None'
-        ),
-        subsets:
-          destinationRule.spec.subsets && destinationRule.spec.subsets.length > 0
-            ? this.generateSubsets(destinationRule.spec.subsets)
-            : 'None',
-        host: destinationRule.spec.host ? <DetailObject name="" detail={destinationRule.spec.host} /> : undefined,
-        createdAt: <LocalTime time={destinationRule.metadata.creationTimestamp || ''} />,
-        resourceVersion: destinationRule.metadata.resourceVersion,
-        actions: this.yamlLink(destinationRule)
-      };
+  noDestinationRules(): IRow[] {
+    return [
+      {
+        cells: [
+          {
+            title: (
+              <EmptyState variant={EmptyStateVariant.full}>
+                <EmptyStateIcon icon={NetworkIcon} />
+                <Title headingLevel="h5" size="lg">
+                  No Destination Rules {!this.props.service.istioSidecar && ' and Istio Sidecar '} found
```

Comment:
Love this empty box. There is one thing I don't understand though. Why are we saying that the sidecar is not set in the Destination Rule tab? It feels like mixing concepts. Bc I understand that the missing istio sidecar icon is still in the page, right?